src/index.ts

```diff
@@ -272,11 +351,10 @@ function extendInterface (
 ) {
   for (const extendedInterface of matchedInterface.getExtends()) {
     const extendedInterfaceName = extendedInterface.compilerNode.expression.getText()
-    const parentNamespace = matchedInterface.getParentNamespace()
-    const matchedExtendedInterface = parentNamespace
-      ? getNamespacedInterface.call(this, parentNamespace, extendedInterfaceName)
-      : getInterface.call(this, interfacePath, extendedInterfaceName)
-
+    const matchedParentNamespace = matchedInterface.getParentNamespace()
+    const parentNamespace = matchedParentNamespace ? matchedParentNamespace : parseDefinitionFiles.call(this, interfacePath)
```

Comment:
This like could be simplified to 
`
matchedInterface.getParentNamespace() || parseDefinitionFiles.call(this, interfacePath)
`